build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -731,21 +731,31 @@ message ExecutedActionMetadata {
 
 // An ActionResult represents the result of an
 // [Action][build.bazel.remote.execution.v2.Action] being run.
+//
+// The server MUST NOT make any inlining decisions on his own but should
+// do so when requested by the client. The total serialized message size
+// SHOULD NOT exceed 1 MiB.
```

Comment:
Where does this 1MiB limit come from? A gRPC message size limit?

Reply:
See commit message :-). It's not a limit but advice from protobuf https://developers.google.com/protocol-buffers/docs/techniques#large-data

Reply:
I'd prefer to drop this recommendation, personally - we use >1MB messages all over the place today, and have seen no issues with doing so. My read is also that it wasn't intended to be a hard limit - the linked doc reads more like guidance not to use 100MB messages than it does a concrete recommendation that e.g. 4MB would be too much.

gRPC itself provides an upper bound on the size of messages - 4MB by default, if I remember correctly. I don't think we need a documented recommendation beyond that.